server/services/socket.js

```diff
@@ -12,14 +12,22 @@ exports.setupSocket = (server, session) => {
         if (socket.handshake.session.user) {
             sockets = sockets.filter(otherSocket => otherSocket.handshake.session.user !== socket.handshake.session.user);
             sockets.push(socket);
-            emitPlayers(gameLogic.getPlayers());
+            emitPlayers(gameLogic.getPlayers())
         }
-    });
-    io.on('disconnect', function (disconnected) {
-        sockets = sockets.filter(socket => socket !== disconnected);
+        socket.on('disconnect', function (disconnected) {
+            sockets = sockets.filter(socket => socket !== disconnected);
+            console.log("Socket with id: " + socket.id + " just disconnected");
+        });
     });
 }
 
+// Create a room
+exports.createRoom = (username, roomId) => {    
+    let socket = sockets.find(socket => socket.handshake.session.user === username);
+    socket.join(`room-${roomId}`);
+    emitPlayersInRoom(roomId, gameLogic.getPlayers());
```

Comment:
Will this currently emit every player?

Reply:
Yeah, again don't want to add any more functionality for this PR, cause it was just for being able to create a room. I used the emitting of players to test that it's actually working. Once I set up gameLogic to be a class, I'll make it emit the correct players only

Reply:
That's cool